server/routes.go

```diff
@@ -242,12 +241,12 @@ func (s *Server) GenerateHandler(c *gin.Context) {
 				res.LoadDuration = checkpointLoaded.Sub(checkpointStart)
 
 				if !req.Raw {
-					tokens, err := r.Tokenize(c.Request.Context(), prompt+sb.String())
+					tokens, err := r.Tokenize(c.Request.Context(), prompt + sb.String())
```

Comment:
gofmt will remove these spaces to to reduce churn, please remove them
```suggestion
					tokens, err := r.Tokenize(c.Request.Context(), prompt+sb.String())
```